includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -686,7 +686,7 @@ public function get_product() {
 			}
 		}
 
-		return false;
+		return null;
```

Comment:
Should we also update the return type in doc comments for this method?

Reply:
No, we shouldn't. The `wc_get_product` method returns `WC_Product|false|null`.